src/components/ResourceCard/AddResource/ResourceActivityType.js

```diff
@@ -44,7 +44,7 @@ let ResourceActivityType = (props) => {
         <div className="activity-item">
           <div
             className="activity-img"
-            style={{ backgroundImage: `url(${activityType.image})`, }}
+            style={{ backgroundImage: `url(${global.config.resourceUrl}${activityType.image})` }}
```

Comment:
I think you need to add another case for pexels here

Reply:
these are static images , no need of pexel here